Godeps/_workspace/src/k8s.io/kubernetes/pkg/runtime/deep_copy_generator.go

```diff
@@ -0,0 +1,609 @@
+/*
```

Comment:
> Why?

At the time?  Time.  The new deep copy generator had multiple bad assumptions about package availability, the old deep copy generator worked, and 1.2 was going to be done two days after I finished this so I wanted reviews, merge, then fix-up.